### ISSUE TITLE
mac falcon bug

### ISSUE THREAD

**Issue by rfyiamcool** [falconry/falcon#628]
[ruifengyun@devops github ]$ pip freeze|grep falcon
falcon==0.3.0
[ruifengyun@devops github ]$ python f.py
Traceback (most recent call last):
  File "f.py", line 1, in <module>
    import falcon
  File "/Library/Python/2.7/site-packages/falcon/__init__.py", line 32, in <module>
    from falcon.api import API, DEFAULT_MEDIA_TYPE  # NOQA
  File "/Library/Python/2.7/site-packages/falcon/api.py", line 22, in <module>
    from falcon.request import Request, RequestOptions
  File "/Library/Python/2.7/site-packages/falcon/request.py", line 35, in <module>
    from six.moves.http_cookies import SimpleCookie
ImportError: No module named http_cookies

**Comment by rfyiamcool** [comment]
code

```
import falcon


class ThingsResource:
    def on_get(self, req, resp):
        """Handles GET requests"""
        resp.status = falcon.HTTP_200  # This is the default status
        resp.body = ('\nTwo things awe me most, the starry sky '
                     'above me and the moral law within me.\n'
                     '\n'
                     '    ~ Immanuel Kant\n\n')

# falcon.API instances are callable WSGI apps
app = api = falcon.API()

# Resources are represented by long-lived class instances
things = ThingsResource()

# things will handle all requests to the '/things' URL path
api.add_route('/things', things)
```

**Comment by jmvrbanac** [comment]
@rfyiamcool Thanks for reporting this. I believe we merged a fix for this in #622 , it just hasn't been released yet. Does the issue you're seeing still show up if you install Falcon from the master repository branch?

**Comment by soso25** [comment]
Hello,

it 's been almost a day I try to run my function "regina.py", but I can not get this makes me errors like this:

import regina.py
Traceback (most recent call last):
File "", line 1, in 
File "/Applications/PyCharm CE.app/Contents/helpers/pydev/pydev_import_hook.py", line 21, in do_import
module = self.system_import(name, args, *kwargs)
File "/Users/Sokhna/Documents/mgp-test-i11vf/regina.py", line 13, in 
import diane
File "/Applications/PyCharm CE.app/Contents/helpers/pydev/pydev_import_hook.py", line 21, in do_import
module = self.system_import(name, args, *kwargs)
File "/Users/Sokhna/Documents/mgp-test-i11vf/diane.py", line 19, in 
import nayru
File "/Applications/PyCharm CE.app/Contents/helpers/pydev/pydev_import_hook.py", line 21, in do_import
module = self._system_import(name, args, *kwargs)
File "/Users/Sokhna/Documents/mgp-test-i11vf/nayru.py", line 1, in 
import falcon
File "/Applications/PyCharm CE.app/Contents/helpers/pydev/pydev_import_hook.py", line 21, in do_import
module = self._system_import(name, args, *kwargs)
File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/falcon/__init.py", line 32, in 
from falcon.api import API, DEFAULT_MEDIA_TYPE # NOQA
File "/Applications/PyCharm CE.app/Contents/helpers/pydev/pydev_import_hook.py", line 21, in do_import
module = self._system_import(name, args, *kwargs)
File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/falcon/api.py", line 18, in 
from falcon import api_helpers as helpers
ImportError: cannot import name api_helpers
I just started python and there I am disappointed because I don't understand anything.

Can anyone help me please :( :( :(

**Comment by kgriffs** [comment]
@soso25 Just to be sure this isn't related to PyCharm, are you able to reproduce the problem by opening up a terminal command line and executing your app directly? If you have a module file that declares an instance of `falcon.API()`, you can self-host it to easily for a quick test, per this example:

https://gist.github.com/kgriffs/32768653b9173df00047

**Comment by soso25** [comment]
Thank you @kgriffs I think it's due to the Mac OS X  because I've fixed it by using Pycharm on Windows 8.1 and also changing Python version. On my Mac I have the 2.7.10 version. Until now, It seems to be OK with the 2.7.11 version on Windows.

Thanks
